plugins/core/jetbrains-community/resources/telemetryOverride.json

```diff
@@ -709,6 +709,93 @@
                     "required": false
                 }
             ]
+        },
+        {
+            "name": "codewhisperer_clientComponentLatency",
```

Comment:
common?

Reply:
yea i'm waiting for approval https://github.com/aws/aws-toolkit-common/pull/738 lol

Reply:
will update when it's merged

Reply:
donedone